src/Vlingo.Xoom.Streams.Tests/Vlingo.Xoom.Streams.Tests.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why do you need that ?

Reply:
Yeah, I get a build failed from AppVeyor suggested `please add xmlns="http://schemas.microsoft.com/developer/msbuild/2003" to the <Project> element. If the project has been authored in the old 1.0 or 1.2 format, please convert it to MSBuild 2003 format.`

Reply:
Yeah, I compared with the other projects and it's not used, I will remove it..

Reply:
I've added the necessary configuration it should build without the namespace. Please merge it into this PR

Reply:
OK, thanks..